pkg/security/security.go

```diff
@@ -42,6 +42,15 @@ const (
 	// DefaultRootCertFilePath is the well-known path for an existing root certificate file
 	DefaultRootCertFilePath = "./etc/certs/root-cert.pem"
 
+	// GkeWorkloadCertChainFilePath is the well-known path for the GKE workload certificate chain file
+	GkeWorkloadCertChainFilePath = "/var/run/secrets/workload-spiffe-credentials/certificates.pem"
```

Comment:
would be nice to have a link to the gke docs for this?

Reply:
Added a link. Quoted from https://cloud.google.com/traffic-director/docs/security-proxyless-setup#create-service:
"On creation, each Pod gets a volume at /var/run/secrets/workload-spiffe-credentials."